sigle/src/icons/Discord.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
@gregogun just saw `@radix-ui/react-icons` published a new release, can you update the package and see if it's working to import it from there?